clang/lib/DPCT/CallExprRewriterMath.cpp

```diff
@@ -636,41 +636,30 @@ std::optional<std::string> MathSimulatedRewriter::rewrite() {
           MigratedArg0 = "(double)(" + MigratedArg0 + ")";
       }
     }
+    auto &SM = DpctGlobalInfo::getSourceManager();
+    std::string Indent =
+        getIndent(SM.getExpansionLoc(Call->getBeginLoc()), SM).str();
```

Comment:
I think keeping the original migration is better.
And the new migration can only be enabled if --optimized-migration is specified. We can use MULTI_STMTS_FACTORY_ENTRY in the rewriter.